authors/apps/articles/urls.py

```diff
@@ -1,17 +1,41 @@
 from django.urls import path
 
 from .views import (
-    ArticleView, ArticleRetrieveUpdateDelete, ShareFacebookView,
-    ShareTwitterView, ShareEmailView
-)
+    ArticleView,
+    ArticleRetrieveUpdateDelete,
+    ShareFacebookView,
+    ShareTwitterView,
+    ShareEmailView,
+    RatingCreateAPIView,
+    RatingRetrieveDestroy)
 
 urlpatterns = [
-    path('articles/', ArticleView.as_view(), name='create_article'),
-    path('articles/<slug>', ArticleRetrieveUpdateDelete.as_view(), name='get_article'),
-    path('articles/<slug>/share/facebook',
-         ShareFacebookView.as_view(), name='share_facebook'),
-    path('articles/<slug>/share/twitter',
-         ShareTwitterView.as_view(), name='share_twitter'),
-    path('articles/<slug>/share/email',
-         ShareEmailView.as_view(), name='share_email'),
+    path(
+        'articles/',
+        ArticleView.as_view(),
+        name='create_article'),
+    path(
+        'articles/<slug>',
+        ArticleRetrieveUpdateDelete.as_view(),
+        name='get_article'),
+    path(
+        'articles/<slug>/share/facebook',
+        ShareFacebookView.as_view(),
+        name='share_facebook'),
+    path(
+        'articles/<slug>/share/twitter',
+        ShareTwitterView.as_view(),
+        name='share_twitter'),
+    path(
+        'articles/<slug>/share/email',
+        ShareEmailView.as_view(),
+        name='share_email'),
+    path(
+        'articles/rate/<slug>/<score>/',
```

Comment:
@ManuelDominic , lets have this URL for rating an article 

```
PUT articles/<slug>/rate
```
With the following request body

```
{
"rating":"<value>"
}
```
And a response of the article that has been rated
A user should-not be able to delete a rating. They should only be able to update.

Use the same url to get my rating of an article i.e

```
GET articles/<slug>/rate
```
Response

```
'article':{
'rating':'<my_rating>'
}
```
Also attach an individual rating when returning an article so that i can know the average rating and my personal rating when retrieving articles